src/gui/previewselectdialog.cpp

```diff
@@ -108,6 +101,7 @@ PreviewSelectDialog::PreviewSelectDialog(QWidget *parent, const BitTorrent::Torr
     m_previewListModel->sort(NAME);
     m_ui->previewList->header()->setContextMenuPolicy(Qt::CustomContextMenu);
     m_ui->previewList->header()->setSortIndicator(0, Qt::AscendingOrder);
+    m_ui->previewList->header()->setFirstSectionMovable(true);
```

Comment:
Nitpick, since there is no noticeable order on those function calls, lets sort them alphabetically.

Reply:
Done.